doc/RELEASE_NOTES.md

```diff
@@ -0,0 +1,828 @@
+
+ * Support for RHEL 9 & CentOS 9
+ * Tool registration kept local to the host where registration happens
+ * Support of Prometheus and PCP tool data collection
+ * The default tool set has been reduced to `iostat`, `sar`, & `vmstat` tools
+ * Removal of gratuitous manipulation of networking firewalls
+ * Removal of gratuitous software installation, only checks for requirements
+   * True for both tools and benchmark script requirements
+   * Change to check command versions instead of RPM versions for `pbench-fio`, `pbench-linpack`, and `pbench-uperf`
+ * The `pbench-linpack` benchmark script now provides result graphs, JSON data files, and supports execution on one or more local / remote hosts
+ * Required use of `--user` with `pbench-move-results`/`pbench-copy-results`
+ * Support for the new HTTP PUT method of posting tar balls
+ * Removal of the dependency on the SCL (Software Collections Library)
+ * Support for `pbench-trafficgen` benchmark script dropped entirely
+ * Deprecation announcements for unused benchmark convenience scripts:
+   * `pbench-run-benchmark`, `pbench-cyclictest`, `pbench-dbench`, `pbench-iozone`, `pbench-migrate`, and `pbench-netperf`
+ * Many, many, bug fixes and behavioral improvements
+
+You can review the [**Full ChangeLog**](https://github.com/distributed-system-analysis/pbench/compare/b0.69-bp...v0.71.0-alpha.0) on GitHub (all 550+ commits, tags `b0.69-bp` to `v0.71.0-alpha.0`), or read a summary with relevant details below.
+
+We did not bump the "major" release version number with these changes because we still don't consider all the necessary functionality in place for such a major version bump.
+
+Note that work on the `v0.71` release started in earnest with the `v0.69.3-agent` release (tagged as `b0.69-bp`).  A number of bug fixes and behaviors from the `v0.71` work have already been back-ported and delivered in the various `v0.69.*` releases since then.  These release notes will highlight only the behavioral changes that have not been back-ported previously.
+
+
+Installation
+====
+
+There are no other installation changes in this release: see the [Getting Started Guide](https://distributed-system-analysis.github.io/pbench/start.html) for how to install or update.
+
+After installation or update, you should have version `0.71.0-XXgXXXXXXXXX` of the `pbench-agent` RPM installed.
+
+RPMs are available from [Fedora COPR](https://copr.fedorainfracloud.org/coprs/portante/pbench/), covering Fedora 35, 36, EPEL 7, 8, 9.
+
+There are Ansible [playbooks](https://galaxy.ansible.com/pbench/agent) available via Ansible Galaxy to install the `pbench-agent`, and the pieces needed (key and configuration files) to be able to send results to a server.  To use the RPMs provided above via COPR with the [playbooks](https://galaxy.ansible.com/pbench/agent), an inventory file needs to include the `fedoraproject_username` variable set to `portante`, for example:
+
+```
+...
+
+[servers:vars]
+fedoraproject_username: portante
```

Comment:
As mentioned in the chat, the setting of variables in the inventory file follows `ini`-file conventions, so it needs an `=` instead of a `:`. Also, the default COPR repo in the ansible roles is `pbench`, whereas for testing we need `pbench-test`, so that needs an additional setting:

```
...
pbench_repo_name = pbench-test
...
```